src/styles/blocks/stars.css

```diff
@@ -0,0 +1,31 @@
+.stars {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0;
+  margin: 0;
+}
+  
+.stars__block {
+  display: flex;
+}
+  
+.stars__review {
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  padding-left: 17px;
+}
+    
+.stars__star {
+  background-image: url(../../images/star.svg);
+  width: 16px;
+  height: 16px;
+  background-position: center;
+  background-repeat: no-repeat;
+  margin-right: 4px;
+}
+    
+.stars__star:nth-child(-n + 4) {
```

Comment:
The code should work for any number of stars if I change the selector to devtools. Use the solution from the previous task on the stars (use the cycle for practice)